Gruntfile.js

```diff
@@ -531,6 +531,12 @@ module.exports = function(grunt) {
           'sentinel/tests/**/*.js'
         ],
       },
+      integration: {
+        src: 'tests/integration/**/*.js',
+        options: {
+          require: 'tests/integration-setup.js'
```

Comment:
Interesting!  In other mocha suites I think we just import chai separately at the top of each test.  This could be a neat alternative, or might confuse people :¬)

I guess this is important for `chai-exclude` though, as otherwise the `chai.use()` call would probably leak from one test to another.  Which would be even more confusing.

Reply:
Turns out I need to use a different option that isn't available through the `grunt-mocha` plugin so I have to use `exec` anyway 👎